### PR TITLE
phpmd configured

### PR DESCRIPTION
**Pull request by ryomaruo182**
test

### CODE REVIEW

**Comment by ryomaruo182** [comment]
このコメント欄見にくいな

**Comment by ryomaruo182** [comment]
プルリクの通知ってちゃんと行ってますか～

**Comment by ryomaruo182** [comment]
ｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱｱ

**Comment by madokasugita** [comment]
@ryomaruo182 暴れてるなｗメール来てました！

File: check_syntax.sh [hunk 1]
```diff
@@ -13,6 +13,7 @@ if [ -n "$CI_PULL_REQUEST" ]; then
     git diff --name-only origin/develop \
         | grep -e '.php$' \
         | xargs vendor/bin/phpcs -n --standard=PSR2 --report=checkstyle \
+        | xargs -I{} vendor/bin/phpmd {} text rules/phpmd_rules.xml \
```
Comment (madokasugita) [inline check_syntax.sh]: ここに追加してもらっていますが、ここだとファイルではなくて引数としてphpcsの結果レポートが入ってきてしまうのでphpmdの分はphpcsとは分離して書くようにお願いしますー(*_ _)

**Review by madokasugita — CHANGES_REQUESTED** [review]

File: check_syntax.sh (hunk 1 above)
Comment (ryomaruo182) [reply check_syntax.sh]: ありがとうございます。
そうですよね(~o~)
修正いたします。

File: controllers/global.php [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (madokasugita) [inline controllers/global.php:1]: ERROR: Default timezone is required since PHP 5.4

File: controllers/global.php [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+global $test1, $test2, $test3;
+global $testXXX;
```
Comment (madokasugita) [inline controllers/global.php:4]: ERROR: Expected 1 blank line at end of file; 2 found

File: controllers/global.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<?php
```
Comment (madokasugita) [inline controllers/global.php:1]: ERROR: Default timezone is required since PHP 5.4

File: controllers/global.php [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+global $test1, $test2, $test3;
+global $testXXX;
+
+global $test4, $test5;
+global $testN;
+
+
+global $test4, $test5;
+global $testN;
+echo "globalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobalglobal";
```
Comment (madokasugita) [inline controllers/global.php:12]: ERROR: Expected 1 newline at end of file; 0 found

File: core/ClassLoader.php [hunk 6]
```diff
@@ -7,6 +7,9 @@
  */
 class ClassLoader
 {
+    echo $_POST['f'];
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$_POST"

File: core/ClassLoader.php [hunk 7]
```diff
@@ -7,6 +7,9 @@
...
+    $auto_q_no_params
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$auto_q_no_params"

File: core/Controller.php [hunk 8]
```diff
@@ -7,6 +7,9 @@
  */
 abstract class Controller
 {
+    echo $_POST['f'];
```
Comment (madokasugita) [inline core/Controller.php]: ERROR: Visibility must be declared on property "$_POST"

File: core/Controller.php [hunk 9]
```diff
@@ -7,6 +7,9 @@
...
+    $auto_q_no_params
```
Comment (madokasugita) [inline core/Controller.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Controller.php]: ERROR: Visibility must be declared on property "$auto_q_no_params"

File: core/ClassLoader.php [hunk 10]
```diff
@@ -7,6 +7,9 @@
  */
 class ClassLoader
 {
+    echo $_POST['f'];
+    $auto_q_no_para
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$auto_q_no_para"

File: core/Controller.php [hunk 11]
```diff
@@ -7,6 +7,9 @@
  */
 abstract class Controller
 {
+    echo $_POST['f'];
+    $auto_q_no_
```
Comment (madokasugita) [inline core/Controller.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Controller.php]: ERROR: Visibility must be declared on property "$auto_q_no_"

File: core/Controller.php [hunk 12]
```diff
@@ -147,8 +147,7 @@ protected function generateCsrfToken($form_name)
      * @param string $token
      * @return boolean
      */
-    protected function checkCsrfToken($form_name, $token)
-    {
+    protected function checkCsrfToken($form_name, $token){
```
Comment (madokasugita) [inline core/Controller.php:150]: ERROR: Opening brace should be on a new line

File: core/Application.php [hunk 13]
```diff
@@ -12,6 +12,7 @@
     protected $response;
     protected $session;
     protected $db_manager;
+    echo $miss_variable
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variable"

File: core/ClassLoader.php [hunk 14]
```diff
@@ -7,7 +7,9 @@
  */
 class ClassLoader
 {
+    
     protected $dirs;
+    echo $miss_variable2
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$miss_variable2"

File: core/Application.php [hunk 15]
```diff
@@ -12,6 +12,10 @@
     protected $response;
     protected $session;
     protected $db_manager;
+//    以下エラーコード
+    echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 16]
```diff
@@ -12,6 +12,10 @@
...
+echo $miss_variable
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/Application.php [hunk 17]
```diff
@@ -12,6 +12,10 @@
...
+echo $miss_variable
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: phpmd_trans.php [hunk 18]
```diff
@@ -0,0 +1,57 @@
+<?php
```
Comment (madokasugita) [inline phpmd_trans.php:1]: ERROR: Default timezone is required since PHP 5.4

File: core/Application.php [hunk 19]
```diff
@@ -12,6 +12,11 @@
...
+echo $miss_variable
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/Application.php [hunk 20]
```diff
@@ -12,6 +12,12 @@
...
+echo $miss_variable
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/Application.php [hunk 21]
```diff
@@ -12,6 +12,13 @@
...
+echo $miss_variable
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/Application.php [hunk 22]
```diff
@@ -11,7 +11,14 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected 4 spaces, found 28

File: core/Application.php [hunk 23]
```diff
@@ -11,7 +11,14 @@
...
+//    以下エラーコード
+    echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token

Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token

Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 24]
```diff
@@ -11,7 +11,16 @@
...
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 25]
```diff
@@ -11,7 +11,14 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 26]
```diff
@@ -11,7 +11,14 @@
...
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/Application.php [hunk 27]
```diff
@@ -11,7 +11,16 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 28]
```diff
@@ -11,7 +11,16 @@
...
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: phpmd_trans.php [hunk 29]
```diff
@@ -0,0 +1,60 @@
...
+        if (preg_match("/<error\sfilename=\"(.*)\"\smsg=\"(.*):.*line:\s([0-9]*),\scol:\s([0-9]*).*/", $line, $matches)) {
+            echo $line;
+            if ($before_file != $matches[1]) {
+                if (!$first_flg) {
+                    $first_flg = 1;
+                    echo <<< HTML
+    <file name="{$matches[1]}">
+
+HTML;
+                }
```
Comment (madokasugita) [inline phpmd_trans.php]: ERROR: Line indented incorrectly; expected 0 spaces, found 16

Comment (madokasugita) [inline phpmd_trans.php]: ERROR: Closing brace indented incorrectly; expected 0 spaces, found 16

File: phpmd_trans.php [hunk 30]
```diff
@@ -0,0 +1,60 @@
...
+HTML;
+            }
+            echo <<< HTML
+        <error line="{$matches[3]}" column="{$matches[4]}" severity="error" message="{$matches[2]}" source="PHP Mess Detector"/>
+
+HTML;
+            $before_file = $matches[1];
+        }
+    }
+    if ($first_flg) {
```
Comment (madokasugita) [inline phpmd_trans.php:38]: ERROR: Line indented incorrectly; expected 0 spaces, found 4

File: phpmd_trans.php [hunk 31]
```diff
@@ -0,0 +1,60 @@
...
+        echo <<< HTML
+    </file>
+
+HTML;
+    }
```
Comment (madokasugita) [inline phpmd_trans.php]: ERROR: Line indented incorrectly; expected 0 spaces, found 4

File: core/Application.php [hunk 32]
```diff
@@ -11,7 +11,18 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 33]
```diff
@@ -11,7 +11,18 @@
...
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/Application.php (hunk 32 above)
Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token

File: core/Application.php [hunk 34]
```diff
@@ -11,7 +11,20 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 35]
```diff
@@ -11,7 +11,20 @@
...
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/ClassLoader.php [hunk 36]
```diff
@@ -1,13 +1,32 @@
...
+
+// ここまで
+
 /**
  * ClassLoader.
  *
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 class ClassLoader
+class ClassLoader
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/ClassLoader.php [hunk 37]
```diff
@@ -1,13 +1,32 @@
...
 {
+    
     protected $dirs;
+    echo $miss_variable2
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$miss_variable2"

File: core/Application.php (hunk 34 above)
Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token: echo,

File: core/Application.php [hunk 38]
```diff
@@ -11,7 +11,22 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 39]
```diff
@@ -11,7 +11,22 @@
...
+                        echo $miss_variablec
+                        echo $miss_variablec
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/ClassLoader.php [hunk 40]
```diff
@@ -1,13 +1,34 @@
...
+
+// ここまで
+
 /**
  * ClassLoader.
  *
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 class ClassLoader
+class ClassLoader
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/ClassLoader.php [hunk 41]
```diff
@@ -1,13 +1,34 @@
...
 {
+    
     protected $dirs;
+    echo $miss_variable2
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$miss_variable2"

File: core/Application.php (hunk 38 above)
Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token: echo,

File: core/Application.php [hunk 42]
```diff
@@ -11,7 +11,23 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 43]
```diff
@@ -11,7 +11,23 @@
...
+                        echo $miss_variablec
+                        echo $miss_variablec
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/ClassLoader.php [hunk 44]
```diff
@@ -1,13 +1,35 @@
...
+
+// ここまで
+
 /**
  * ClassLoader.
  *
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 class ClassLoader
+class ClassLoader
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/ClassLoader.php [hunk 45]
```diff
@@ -1,13 +1,35 @@
...
 {
+    
     protected $dirs;
+    echo $miss_variable2
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$miss_variable2"

File: core/Application.php (hunk 42 above)
Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token: echo,

File: core/Application.php [hunk 46]
```diff
@@ -11,7 +11,25 @@
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 47]
```diff
@@ -11,7 +11,25 @@
...
+                        echo $miss_variablec
+                        echo $miss_variablec
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/ClassLoader.php [hunk 48]
```diff
@@ -1,13 +1,37 @@
...
+
+// ここまで
+
 /**
  * ClassLoader.
  *
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 class ClassLoader
+class ClassLoader
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/ClassLoader.php [hunk 49]
```diff
@@ -1,13 +1,37 @@
...
 {
+    
     protected $dirs;
+    echo $miss_variable2
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$miss_variable2"

File: core/Application.php (hunk 46 above)
Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token: echo,

File: core/Application.php [hunk 50]
```diff
@@ -6,12 +6,32 @@
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 abstract class Application
+abstract class Application
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/Application.php [hunk 51]
```diff
@@ -6,12 +6,32 @@
...
 {
     protected $debug = false;
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 52]
```diff
@@ -6,12 +6,32 @@
...
+                        echo $miss_variablec
+                        echo $miss_variablec
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/ClassLoader.php [hunk 53]
```diff
@@ -1,13 +1,38 @@
...
+
+// ここまで
+
 /**
  * ClassLoader.
  *
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 class ClassLoader
+class ClassLoader
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/ClassLoader.php [hunk 54]
```diff
@@ -1,13 +1,38 @@
...
 {
+    
     protected $dirs;
+    echo $miss_variable2
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$miss_variable2"

File: core/Application.php (hunk 50 above)
Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token: abstract,

File: core/Application.php [hunk 55]
```diff
@@ -6,12 +6,34 @@
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 abstract class Application
+abstract class Application
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/Application.php [hunk 56]
```diff
@@ -6,12 +6,34 @@
...
 {
     protected $debug = false;
     protected $request;
     protected $response;
     protected $session;
-    protected $db_manager;
+                            protected $db_manager;
+//    以下エラーコード
+                        echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/Application.php]: ERROR: Visibility must be declared on property "$miss_variablec"

File: core/Application.php [hunk 57]
```diff
@@ -6,12 +6,34 @@
...
+                        echo $miss_variablec
+                        echo $miss_variablec
+echo $miss_variablec
```
Comment (madokasugita) [inline core/Application.php]: ERROR: Line indented incorrectly; expected at least 4 spaces, found 0

File: core/ClassLoader.php [hunk 58]
```diff
@@ -1,13 +1,41 @@
...
+
+// ここまで
+
 /**
  * ClassLoader.
  *
  * @author Katsuhiro Ogawa <fivestar@nequal.jp>
  */
 class ClassLoader
+class ClassLoader
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Each class must be in a namespace of at least one level (a top-level vendor name)

File: core/ClassLoader.php [hunk 59]
```diff
@@ -1,13 +1,41 @@
...
 {
+    
     protected $dirs;
+    echo $miss_variable2
```
Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: There must not be more than one property declared per statement

Comment (madokasugita) [inline core/ClassLoader.php]: ERROR: Visibility must be declared on property "$miss_variable2"

File: core/Application.php (hunk 55 above)
Comment (madokasugita) [inline core/Application.php]: ERROR: Unexpected token: abstract

File: controllers/global.php [hunk 60]
```diff
@@ -0,0 +1,12 @@
+<?php
```
Comment (madokasugita) [inline controllers/global.php:1]: ERROR: Default timezone is required since PHP 5.4

File: controllers/global.php (hunk 5 above)
Comment (madokasugita) [inline controllers/global.php:12]: ERROR: Expected 1 newline at end of file; 0 found

File: core/Controller.php (hunk 12 above)
Comment (madokasugita) [inline core/Controller.php:150]: ERROR: Opening brace should be on a new line

File: phpmd_trans.php [hunk 61]
```diff
@@ -0,0 +1,50 @@
+<?php
```
Comment (madokasugita) [inline phpmd_trans.php:1]: ERROR: Default timezone is required since PHP 5.4

**Review by madokasugita — APPROVED** [review]

**Pull request merged**